scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -43,6 +43,24 @@ class FormatWriter(formatOps: FormatOps) {
                 .getOrElse(token.syntax, token.syntax)
             sb.append(rewrittenToken)
         }
+
+        if (runner.dialect.allowTrailingCommas &&
```

Comment:
Let's move this to a helper method `handleTrailingComma` or something like that

Reply:
I agree, I was about to do it!